OpenSim/Simulation/Test/testRealizeDynamics.cpp

```diff
@@ -0,0 +1,90 @@
+/* -------------------------------------------------------------------------- *
+ *                    OpenSim:  testRealizeDynamics.cpp                       *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2012 Stanford University and the Authors                *
+ * Author(s): Peter Eastman, Ajay Seth                                        *
```

Comment:
> Do you think that this test is a good idea or should I remove it? 

I think it's a *great* idea! But it does need to have the expected answers built in so it will throw an exception if something is wrong and run silently otherwise. We have test macros for that stuff.